docs/md/views/3dngl.md

```
@@ -0,0 +1,98 @@
+* Show Distance Labels - displayed cross-links are annotated with a small label showing their length in Angstroms. These show up regardless for selected or highlighted cross-links.
+* Protein Chain Label Style - Long, Short, None, Fixed Size - a set of radio buttons for deciding the content of the label annotating each protein. Long for a verbose description which NGL pulls from the PDB file (plus the short description), short for a label with the protein name and chain index, and none to remove these labels altogether. A final checkbox, "Fixed Size", determines whether these labels stay a fixed size when the structure is zoomed in or out.
+
+Finally, the "Export" dropdown offers three options for exporting various parts of the PDB/cross-links combinations.
+
+* PDB & Cross-Links - Exports a basic copy of the PDB file (ATOM records only) with a number of LINK and CONECT files that represent cross-links that can be fully plotted within the structure. Since this is a PDB file, there is a limit of 100,000 atoms and structures with more than one model (NGL fuses PDBs by giving them different model indices) cannot have their inter-model cross-links successfully exported. This can be used in various molecule viewers but usually requires trial and error to figure out how to get the cross-links to appear in each one.
+
+* Pymol Command File - Exports a Pymol command file that will reload and render the cross-links in the Pymol viewer. This supports multiple PDB files and inter-model/PDB cross-links, but obviously is limited to Pymol only.
+
+* Haddock Distance Restraints File (not definitive)- for use with the [HADDOCK](https://haddock.science.uu.nl/services/HADDOCK2.2/) structure docker, this exports all the inter-model cross-links in a format compatible with the HADDOCK tool. These inter-model cross-links tend to be those we know exist between two pdb files, but since we cannot be sure of the PDB's relative orientation their distances are meaningless. However, HADDOCK can use their existence and restrictions based on cross-linker length to calculate the correct (or near to) orientation.
+* csv export file - a plain csv file with crosslinks according to the residue numbering and Chain IDs of the pdb file.
+* Chimerax Pseudobonds file - for visualizing crosslinks in [UCSF ChimeraX ](https://www.rbvi.ucsf.edu/chimerax/).
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside link text

(MD039, no-space-in-links)